src/test/java/org/wildfly/security/sasl/test/StringPrepTest.java

```diff
@@ -837,16 +837,16 @@ public void testForbidUnassigned() throws Exception {
     // ---------------------- helpers ----------------------
 
     private String codePointToString(int codePoint) {
-        ByteStringBuilder b = new ByteStringBuilder();
-        b.appendUtf8Raw(codePoint);
-        return new String(b.toArray(), StandardCharsets.UTF_8);
+        StringBuffer sb = new StringBuffer();
```

Comment:
You are true, I didn't know about this solution. In that case the helper codepointToString is not useful and I remove it.